lib/matplotlib/tests/test_usetex.py

```diff
@@ -12,24 +12,39 @@
     pytestmark = pytest.mark.skip('Missing TeX of Ghostscript or dvipng')
 
 
-@image_comparison(baseline_images=['test_usetex'],
-                  extensions=['pdf', 'png'],
-                  tol={'aarch64': 2.868, 'x86_64': 2.868}.get(
-                      platform.machine(), 0.3
-                      ))
+@image_comparison(
+    baseline_images=['test_usetex'],
+    extensions=['pdf', 'png'],
+    tol={'aarch64': 2.868, 'x86_64': 2.868}.get(platform.machine(), 0.3),
```

Comment:
Since you're replacing the file, do you really need this extra tolerance (on `x86_64` at least)?

Reply:
hopefully not.